crates/vhost/src/vhost_user/message.rs

```diff
@@ -500,18 +544,74 @@ impl VhostUserMemoryRegion {
             mmap_offset,
         }
     }
+
+    /// Creates range from Self.
+    pub fn as_range(&self, file: File) -> (GuestAddress, usize, Option<FileOffset>) {
+        let g_addr = GuestAddress(self.guest_phys_addr);
+        let len = self.memory_size as usize;
+        let f_off = FileOffset::new(file, self.mmap_offset);
+
+        (g_addr, len, Some(f_off))
+    }
+
+    /// Creates range from Self.
```

Comment:
This looks like a copy paste error? Overall, I currently have the feeling that most doc strings do not really add too much compared to what the function signature already gives away. Maybe some more detail could be helpful here and there...